pack-extension.sh

```diff
@@ -2,7 +2,12 @@
 
 rm -rf ./dist
 NODE_ENV=production node_modules/.bin/webpack -p
-cp manifest.json icons/* ./dist
+
+echo "Manually removing 'localhost:8000' from the manifest.json"
+# This is savage.
+sed '/localhost:8000/d' manifest.json > ./dist/manifest.json
```

Comment:
yep :crossed_swords: it's a battle there. The right approach is to have a template for `manifest.json`, but we can do it later.

I added a note here #6.